pkg/utils/log.go

```diff
@@ -138,6 +138,15 @@ func (l *Logger) Warning(format string, v ...interface{}) {
 	}
 }
 
+// Log error variable as a warning
+func (l *Logger) Warn(err error) error {
```

Comment:
Awesome.  This needs unit tests.  You may want to name the ones that take error something similar.  For example, we already have `Err()` and `LogError()`, so this could be `WarnErr()` or `WarningError()` or something like that.